### ISSUE TITLE
[BUG] - bad display of direction of CylinderSegment

### ISSUE THREAD

**Issue by OrtnerMichael** [magpylib/magpylib#385]
arrow position is not rotated with magnet!!!

```
import numpy as np
import magpylib as magpy

n = 100
diameter = n/np.pi     # [mm]
polesize = 360/n       # [deg]

wheel = magpy.Collection()
for i in range(n):
    pole = magpy.magnet.CylinderSegment(
        magnetization=(200,0,0),
        dimension=(diameter-2, diameter, 5, -polesize/2, polesize/2))
    wheel + pole
    wheel.rotate_from_angax(polesize, 'z')
wheel.rotate_from_angax(90, 'x')
magpy.display(wheel, zoom=0, style_magnetization_size=0.2)
```

![image](https://user-images.githubusercontent.com/29252289/141271834-1238cb26-1b97-4125-9357-31dbcbfc5098.png)

**Comment by Alexboiboi** [comment]
@OrtnerMichael Can you solve this one?

**Comment by Alexboiboi** [comment]
trivial fix in d162aca5d98802ee0a1e38f239ed660854795d6d
![image](https://user-images.githubusercontent.com/29252289/141272963-53ec4403-7d5c-4c90-a7f6-1c2a24707657.png)
